integration/cluster.go

```diff
@@ -465,6 +465,8 @@ type member struct {
 
 	// serverClient is a clientv3 that directly calls the etcdserver.
 	serverClient *clientv3.Client
+
+	keepDataDirStop bool
```

Comment:
`keepDataDirTerminate bool`? Another option is `Stop` the member and remove from c.Members, before calling removeMember on the ID so it never hits `Terminate`.